hub/integrations/pytorch_old.py

```diff
@@ -21,13 +21,26 @@ def dataset_to_pytorch(
     pin_memory: Optional[bool] = False,
     python_version_warning: bool = True,
 ):
+    global torch
+    try:
+        import torch
+    except ModuleNotFoundError:
+        raise ModuleNotInstalledException(
+            "'torch' should be installed to convert the Dataset into pytorch format"
+        )
+
     dataset.flush()
     pytorch_ds = TorchDataset(
         dataset,
         transform,
         tensors,
         python_version_warning=python_version_warning,
     )
+    # TODO add pytorch for num_workers > 1
+    if num_workers > 0:
+        raise NotImplementedError(
+            "Multiproccessed pytorch training is not support for Python version < 3.8. Please set num_workers equal to 0 or upgrade to python 3.8."
```

Comment:
How does this line _only_ get executed for < 3.8? Is there a branch I'm not seeing?

Reply:
yes, it is here https://github.com/activeloopai/Hub/blob/fix/2.0/chunk-engine/pytorch/hub/integrations/__init__.py